# disable VCS stamping 
# - due to nested repos it is not possible to embed vcs info with github build action
# 
--- Makefile.org	2026-10-22 13:33:56.334158618 +0300
+++ Makefile	2026-10-22 13:34:00.325208122 +0300
@@ -146,7 +146,7 @@
 .PHONY: cloudflared
 cloudflared:
 	$(if $(filter true,$(FIPS)),$(info Building cloudflared with Go's FIPS 140 module))
-	GOOS=$(TARGET_OS) GOARCH=$(TARGET_ARCH) $(ARM_COMMAND) go build -mod=readonly $(GO_BUILD_TAGS) $(LDFLAGS) $(IMPORT_PATH)/cmd/cloudflared
+	GOOS=$(TARGET_OS) GOARCH=$(TARGET_ARCH) $(ARM_COMMAND) go build -buildvcs=false -mod=readonly $(GO_BUILD_TAGS) $(LDFLAGS) $(IMPORT_PATH)/cmd/cloudflared
 
 ifeq ($(FIPS), true)
 	./check-fips.sh $(BUILT_BINARY_PATH)
